setup.sh

```diff
@@ -348,8 +348,14 @@ download_db_dump         # pre-req: install_deps
 create_pg_databases      # pre-req: install_deps
 create_default_keeper_config # pre-req: update_userinfo
 
-# We want to run this only with the brew version of python, NOT OSX's python3
-install_keeper $(brew --prefix)/bin/python3
+# If brew is installed, run this only with the brew version of python, NOT OSX's python3
+if [ -n "$(which brew)" ]; then
```

Comment:
This will print error messages to stderr if brew isn't installed.  Instead, you can do:
```
if which brew >/dev/null 2>&1; then
   install_keeper $(brew --prefix)/bin/python3
else
   install_keeper python3
fi

Reply:
Thanks for the tip! Confirmed the following echoes on my laptop with macOS but not Ubuntu:

`which brew >/dev/null 2>&1 && echo "brew installed"`